tests/functional/base_pages/summary.page.js

```diff
@@ -1,13 +1,17 @@
 import QuestionPage from "./question.page";
 
-class ThankYouPage extends QuestionPage {
+class SummmaryPage extends QuestionPage {
```

Comment:
Is this used?

Reply:
Was only for 1 test but has been removed as part of the base pages refactor.